app/main.py

```diff
@@ -1,25 +1,75 @@
 from abc import ABC
+from typing import Callable
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Callable, name: str) -> None:
+        self.private_name = "_" + name
+
+    def __get__(self, instance: Callable, owner: Callable) -> int:
+        return getattr(instance, self.private_name)
+
+    def __set__(self, instance: Callable, value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError("Input the right data")
+        if not self.min_amount <= value <= self.max_amount:
+            raise ValueError("Parameters must be in right range")
+        return setattr(instance, self.private_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: int, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self) -> None:
+        pass
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/209964541-a7f29a4f-0e39-47c4-9e04-a3ac6fda3e12.png)
fix this,please